go/src/koding/klientctl/config/config.go

```diff
@@ -82,6 +82,11 @@ var (
 	// KontrolURL is overwritten during deploy via linker flag.
 	KontrolURL = "https://koding.com/kontrol/kite"
 
+	// TunnelKiteAddress is the address that koding's tunnel service is run on.
+	//
+	// This is overwritten during deploy via linker flag.
+	TunnelKiteAddress = "http://t.koding.com/kite"
```

Comment:
TunnelKite is t.koding.com for production and dev-t.koding.com for development.

Could you add this url to koding/config/config.json and read it from there? We're going to move all the hardcoded urls and settings to that package either way, the less new ones we add in the meantime the better.